src/config/index.js

```diff
@@ -114,6 +116,7 @@ class Config {
    * clientId and host, or an environment that matches a default audience/environment.
    *
    * @param {CustomAudience} config A custom audience configuration to parse
+   * @param {Object.<string, Audience>} audiences An object with keys for environment names and values of Audience information
```

Comment:
doh, i can't believe i left that off

Reply:
Don't beat yourself up. If it weren't for me using the ultra cool IntelliJ IDEA Ultimate IDE, I'd have never seen the warning that the property was undocumented.